src/main/resources/templates/db_scripts/updateIllPolicyWillNotLend.sql

```diff
@@ -0,0 +1 @@
+UPDATE ${myuniversity}_${mymodule}.ill_policy SET jsonb = jsonb_set(jsonb, '{name}', '"Will not lend"') where id='b0f97013-87f5-4bab-87f2-ac4a5191b489';
```

Comment:
Great work! We usually create a test for a migration, you can use `org.folio.rest.api.ModesOfIssuanceMigrationScriptTest#canMigrateModeOfIssuance` as an example for your case.

Reply:
Test added.